inc/target_actor.class.php

```diff
@@ -92,6 +92,39 @@ public static function getTypeName($nb = 0) {
    }
 
    public function prepareInputForAdd($input) {
+      $requiredKeys = ['itemtype', 'items_id', 'actor_role', 'actor_type'];
```

Comment:
add integrity checks, does not handles all cases for now, but fix the inconsistency found this week